### ISSUE TITLE
Cannot find transitive dependencies for com.google.guava 32.1.0-jre or above on v1.1.1

### ISSUE THREAD

**Issue by mattbutalla** [spring-gradle-plugins/dependency-management-plugin#361]
It looks like the latest release (v1.1.1) of dependency-management-plugin is having trouble with finding transitive dependencies, at least for com.google.guava 32.1.0-jre or above.

Example simplistic build.gradle:
```
plugins {
	id 'java'
	id 'io.spring.dependency-management' version '1.1.1'
}

dependencies {
    implementation 'com.google.guava:guava:32.1.1-jre'
}
```

When running `./gradlew dependencies` I get the following (for compileClasspath):
```
compileClasspath - Compile classpath for source set 'main'.
\--- com.google.guava:guava:32.1.1-jre
     +--- com.google.guava:failureaccess:1.0.1
     +--- com.google.code.findbugs:jsr305 FAILED
     +--- org.checkerframework:checker-qual FAILED
     +--- com.google.errorprone:error_prone_annotations FAILED
     \--- com.google.j2objc:j2objc-annotations FAILED
```

When changing id 'io.spring.dependency-management' version '1.1.1' to `1.1.0` _or_ com.google.guava:guava to `32.0.1-jre` the dependencies are resolved:
```
compileClasspath - Compile classpath for source set 'main'.
\--- com.google.guava:guava:32.0.1-jre
     +--- com.google.guava:failureaccess:1.0.1
     +--- com.google.guava:listenablefuture:9999.0-empty-to-avoid-conflict-with-guava
     +--- com.google.code.findbugs:jsr305:3.0.2
     +--- org.checkerframework:checker-qual:3.33.0
     +--- com.google.errorprone:error_prone_annotations:2.18.0
     \--- com.google.j2objc:j2objc-annotations:2.8
```

Can this be re-produced? Possibly a corporate issue too...

**Comment by mattbutalla** [comment]
Sorry about that @wilkinsona , thanks for all your work

**Comment by wilkinsona** [comment]
No worries. I'd rather hear about it twice than not at all.

**Comment by stijnvancrombrugge** [comment]
We experience the same issues starting from the 1.1.1 release. We have an application which has a platform dependency. The platform on its own has dependencies exposed using `api 'x'`. The nested dependency x was included in our application until version 1.1.0, but since version 1.1.1 the nested exposed dependencies are not included anymore. Only by adding the direct dependency in our application it gets included. We still face this issue in the 1.1.3 release.
